src/imageio/storage/gallery.c

```diff
@@ -104,7 +104,7 @@ static void button_clicked(GtkWidget *widget, dt_imageio_module_storage_t *self)
   gallery_t *d = (gallery_t *)self->gui_data;
   GtkWidget *win = dt_ui_main_window(darktable.gui->ui);
   GtkWidget *filechooser = gtk_file_chooser_dialog_new(
-      _("select directory"), GTK_WINDOW(win), GTK_FILE_CHOOSER_ACTION_SELECT_FOLDER, _("_cancel"),
+      C_("dialog title", "select directory"), GTK_WINDOW(win), GTK_FILE_CHOOSER_ACTION_SELECT_FOLDER, _("_cancel"),
```

Comment:
"title" ?

Reply:
I'd prefer "dialog title" as this makes it unmistakably clear for translators what and where this is. While there probably won't be any confusion with this specific string, I think it's better to use precise terminology, especially as there's no practical downside that I can think of...